games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="An Amazing 3D Flight Simulator"
+DESCRIPTION="The goal of the FlightGear project is to create a sophisticated \
+and open flight simulator framework for use in research or academic environments, \
+pilot training, as an industry engineering tool, for DIY-ers to pursue their favorite \
+interesting flight simulation idea, and last but certainly not least as a fun, \
+realistic, and challenging desktop flight simulator. We are developing a sophisticated, \
+open simulation framework that can be expanded and improved upon by anyone interested \
+in contributing."
+HOMEPAGE="https://www.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flighgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="924bcaec4bead47d01638f1033c0ca95c4a3fff65bd43d9c5b171d354b4b28f6"
+ADDITIONAL_FILES="flightgear.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	flightgear$secondaryArchSuffix
+	cmd:fgcom
+	cmd:fgelev
+	cmd:fgfs
+	cmd:fgjs
+	cmd:fgpanel
+	cmd:fgtraffic
+	cmd:fgviewer
+	cmd:GPSsmooth
+	cmd:JSBSim
+	cmd:js_demo
+	cmd:metar
+	cmd:MIDGsmooth
+	cmd:UGsmooth
+	cmd:yasim
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libqt5$secondaryArchSuffix
+	lib:boost$secondaryArchSuffix
+	lib:plib$secondaryArchSuffix
+	lib:simgearopenal$secondaryArchSuffix
+	lib:sqlite$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:harfbuzz$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libqt5$secondaryArchSuffix
+	devel:plib$secondaryArchSuffix
+	devel:boost$secondaryArchSuffix
+	devel:plib$secondaryArchSuffix
+	devel:simgear$secondaryArchSuffix
+	devel:sqlite$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:make
+	cmd:cmake >= 3.0
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. \
+		-DFG_DATA_DIR:STRING="%{_datadir}/flightgear" \
+		-DENABLE_GPSSMOOTH:BOOL=OFF -DENABLE_FGVIEWER:BOOL=OFF \
+		-DENABLE_FGELEV:BOOL=OFF -DENABLE_METAR:BOOL=OFF \
+		-DSYSTEM_SQLITE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF \
+		-DBUILD_TESTING:BOOL=OFF -DENABLE_TESTS:BOOL=OFF \
+		-DENABLE_JS_DEMO:BOOL=OFF
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+ 	addAppDeskbarSymlink $appsDir/FlightGear
```

Comment:
Do you get something in `$appsDir/` ?
Does the project know that on Haiku it should be installed there? Because there are no indications in the recipe that this is where you want the runtime to be installed.

Reply:
OK, I got it. I guess the runtime gets installed in `$binDir`. In that case you can keep the symlink:
```diff
-	addAppDeskbarSymlink $appsDir/FlightGear
+ 	addAppDeskbarSymlink $binDir/flightgear FlightGear
```
Warning: this is completely untested.